src/Test/L0/Worker/StepsRunnerL0.cs

```diff
@@ -326,11 +327,12 @@ private TestHostContext CreateTestContext([CallerMemberName] String testName = "
                     Assert.Equal(2, variableSet.Length);
                     variableSet[0].Verify(x => x.RunAsync());
                     variableSet[1].Verify(x => x.RunAsync(), Times.Never());
+                    Assert.Equal(TaskResult.Skipped, variableSet[1].Object.ExecutionContext.Result);
```

Comment:
@ericsciple  Skip method is in ExecutionContext, not in Step. We will not be able to call variableSet[1].Verify(x => x.skip()); We should call variableSet[1].Verify(x => x.ExecutionContext.Skip()), but the executioncontext we get here is not a mock object. Instead we should create the mock object and have a reference so that we could verify like stepContext.Verify(x => x.skip()); 
Please take a look at the updated tests.